prog/postgres/postgres_server_nexus.rb

```diff
@@ -275,6 +280,7 @@ def refresh_walg_credentials
 
     walg_config = postgres_server.timeline.generate_walg_config
     vm.sshable.cmd("sudo -u postgres tee /etc/postgresql/wal-g.env > /dev/null", stdin: walg_config)
+    vm.sshable.cmd("sudo tee /usr/lib/ssl/certs/walg.pem > /dev/null", stdin: postgres_server.timeline.blob_storage.certs)
```

Comment:
I'd name this as blob_storage_ca.pem. Also, why it is pem, not crt?

Reply:
Updated as requested